factor/lists.py

```diff
@@ -3,48 +3,78 @@
 # ex: [0, 2, -1, 15] -> 16
 def summation(numbers: list[int]) -> int:
     # TODO implement
-    return 0
+    # relates to zero when it goes to positive and negative number
+    sum=0 
+    for number in numbers:
+         sum += number
+    return sum
 
 # find_negative() accepts a list of integers containing one negative number
 # and returns the *position* of the negative number. You may safely assume
 # the provided list contains only a single negative number.
 # ex: [11, 13, -1, 0, 9] -> 2
 def find_negative(numbers: list[int]) -> int:
     # TODO implement
-    return 0
+    #the list that is given a give a list of negative numbers
+    for index in range (len(numbers)):
+      if numbers [index] < 0 :
+       return index
 
 # remove() accepts a list of integers and an int n. The method removes *all instances*
 # of n from the provided list and returns a new list with no instances of n.
 # ex: [0, 1, 1, 2, 2, 3], n = 2 -> [0, 1, 1, 3]
 def remove(numbers: list[int], n: int) -> list[int]:
     # TODO implement
-    return []
+    #code to take out a certain number out of a number list
+    while n in numbers: 
+        numbers.remove(n)
+    return numbers
+
 
 # merge() accepts two *pre-sorted* lists of integers and returns a new *sorted* list.
 # WARNING do not assume lists are of equal length!
 # ex: [0, 2, 4, 8] + [1, 3, 5] -> [0, 1, 2, 3, 4, 5, 8]
 def merge(first: list[int], second: list[int]) -> list[int]:
     # TODO implement
-    # HINT use list.sort()
-    return []
+    #Mix numbers that are odd but in order
```

Comment:
no capital letters outside of class names, special comments, and grammatically correct text in Python